packages/components/quick-filters/src/quick-filters.readme.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, Markdown } from '@storybook/blocks';
 import ReadMe from './../README.md?raw'
 
-<Meta tags={['local-dev']} title="components/QuickFilters/Readme" />
+<Meta tags={[/*'local-dev'*/]} title="components/QuickFilters/Readme" />
```

Comment:
This is the entire `QuickFilters` component, so there isn't a need to put this back into `local-dev` once the PR is merged